content/java/concepts/errors/terms/invalidclassexception/invalidclassexception.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: 'InvalidClassException'
+Description: 'Occurs when the Serialization runtime detects a problem with a class.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Errors'
+  - 'Error Handling'
+  - 'Exceptions'
+  - 'Debugging'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`InvalidClassException`** is an [exception](https://www.codecademy.com/resources/docs/general/error) that occurs when the [Serialization](https://www.codecademy.com/resources/docs/general/serialization) runtime detects a problem with a [class](https://www.codecademy.com/resources/docs/java/classes). The problem can be one of the following:
```

Comment:
```suggestion
The **`InvalidClassException`** occurs when the [Serialization](https://www.codecademy.com/resources/docs/general/serialization) runtime detects a problem with a [class](https://www.codecademy.com/resources/docs/java/classes). The problem can be one of the following:
```

Since there is a link back to the Java: Errors entry above the entry header, the link in the opening sentence isn't necessary. I also think the link to the General: Error entry can be potentially confusing since this is a Java term entry. For a future PR, we should look to remove these links from the Java: Errors terms that currently exist.

Reply:
Also, I'm not sure how to feel about the link to "Serialization". I understand we have the General entry for it, but isn't this context referring to a more specific, Java-centric term (e.g., "Serialization runtime")?